src/voice.ts

```diff
@@ -99,59 +98,63 @@ export class Voice extends Element {
     this.mode = Voice.Mode.STRICT;
   }
 
-  // Get the total ticks in the voice
+  /** Get the total ticks in the voice */
   getTotalTicks(): Fraction {
     return this.totalTicks;
   }
 
-  // Get the total ticks used in the voice by all the tickables
+  /** Get the total ticks used in the voice by all the tickables */
   getTicksUsed(): Fraction {
     return this.ticksUsed;
   }
 
-  // Get the largest width of all the tickables
+  /** Get the largest width of all the tickables */
```

Comment:
Super duper minor, but I would prefer that sentence comments end with a period.

Reply:
done